### PR TITLE
updating the search scalability

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/8501#issuecomment-759717118

### CODE REVIEW

File: install/perform_scale.adoc [hunk 1]
```diff
@@ -86,8 +86,8 @@ The memory required is proportional to the number of Kubernetes resources and th
 | 878 MB
 |===
 
-By default, the data store is deployed with a memory limit of 1 GB.
-If you are managing larger clusters, you might need to increase this limit by editing the deployment named `search-prod-xxxxx-redisgraph` in the hub cluster namespace.
+By default, the data store is deployed with a memory limit of 4 GB.
+If you are managing larger clusters, you might need to increase this limit by editing the `limit_memory` of the `redisgraph_resource` in the `searchoperator` hub cluster namespace. When the change is made, delete the `search-redisgraph` StatefulSet for the new limit to take effect.
```
Comment (SherinV) [inline install/perform_scale.adoc]: > by editing the limit_memory of redisgraph_resource under searchoperator in the hub cluster namespace.

The edit should be made to searchoperator. An example command to increase memory to 8Gi is shown below:

> oc patch searchoperator searchoperator --type='merge' -p '{"spec":{"redisgraph_resource":{"limit_memory":"8Gi"}}}'

@xrajesh : Please review the command.

Comment (SherinV) [reply install/perform_scale.adoc]: We have to update the [table](https://github.com/open-cluster-management/rhacm-docs/blob/d00c6de3cc5f88c375616d8b673fc11fa7d2efcb/install/perform_scale.adoc#physical-memory) too

Comment (xrajesh) [reply install/perform_scale.adoc]: Verified the command for patch @SherinV

Comment (dockerymick) [reply install/perform_scale.adoc]: Thanks for the live review @SherinV! @xrajesh may you re-review the PR

**Comment by dockerymick** [comment]
Waiting to verify if the changes made in the table also apply to v2.1

**Review by SherinV — APPROVED** [review]

File: install/perform_scale.adoc [hunk 2]
```diff
@@ -63,31 +63,37 @@ The memory required is proportional to the number of Kubernetes resources and th
...
 | 50 medium
 | 250,000
 | 750,000
-| 878 MB
+| 2 Gi
 |===
 
-By default, the data store is deployed with a memory limit of 1 GB.
-If you are managing larger clusters, you might need to increase this limit by editing the deployment named `search-prod-xxxxx-redisgraph` in the hub cluster namespace.
+By default, the data store is deployed with a memory limit of 4 Gi.
```
Comment (SherinV) [inline install/perform_scale.adoc]:
```suggestion
By default, the redisgraph pod (search-redisgraph-0) is deployed with a memory limit of 4 Gi.
```

File: install/perform_scale.adoc [hunk 3]
```diff
@@ -63,31 +63,37 @@ The memory required is proportional to the number of Kubernetes resources and th
...
+If you are managing larger clusters, you might need to increase this limit by editing the `redisgraph_resource.limit_memory` for the `searchoperator` in the hub cluster namespace. For example, you can update the limit to 8Gi with the following command:
+
+----
+oc patch searchoperator searchoperator --type='merge' -p '{"spec":{"redisgraph_resource":{"limit_memory":"8Gi"}}}'
+----
+
+When the change is made, delete the `search-redisgraph` StatefulSet for the new limit to take effect.
 
 [#write-throughput-cache-recovery-time]
 === Write throughput (cache recovery time)
```
Comment (SherinV) [inline install/perform_scale.adoc:98]: We can add this line before the Write throughput table. 
`When the datastore is cleared, we measured recovery times with different number of managed clusters.`

**Review by SherinV — CHANGES_REQUESTED** [review]

File: install/perform_scale.adoc (hunk 3 above)
Comment (SherinV) [reply install/perform_scale.adoc:98]: Updates based on discussion with @xrajesh

Comment (dockerymick) [reply install/perform_scale.adoc:98]: @SherinV are you ok with the following update:

When the datastore is cleared, recovery times are measured for a different number of managed clusters.

Comment (SherinV) [reply install/perform_scale.adoc:98]: What we want the user to know is

> When the datastore is cleared, we measured recovery times with different number of managed clusters and the results are captured in the table below. 

You can change it as needed.

**Pull request merged**